packages/website/src/components/KitItem.astro

```diff
@@ -1,48 +1,17 @@
 ---
-import { ArrowRightIcon } from '../icons/heroicons';
 import TechIcon from './TechIcon.astro';
-import { GitHubIcon } from '../icons/GitHubIcon.tsx';
 const { kit } = Astro.props;
-const technologyStubsCount = kit.technologies.length < 3 ? (3 - kit.technologies.length) : 0;
 ---
 
 <div
-  class="bg dark:bg-gray-800 border border-gray-400 dark:border-gray-700 rounded-lg group hover:bg-brand-500 hover:dark:bg-brand-500 transition-colors overflow-hidden"
+  class=" bg dark:bg-gray-800 border border-gray-400 dark:border-gray-700 rounded-lg group hover:bg-brand-500 hover:dark:bg-brand-500 transition-colors overflow-hidden max-w-[419px] max-h-[541px]"
 >
-  <div class="px-4 py-6 space-y-4">
-    {kit.technologies.map(({ name, Icon, tags }) => (
-      <div class="flex items-center space-x-6">
+<h2 class="text-2xl px-8 pt-6 t-light dark:dark-t group-hover:text-white min-h-[100px]">{kit.name}</h2>
+  <div class="grid grid-cols-3 grid-rows-4 pt-2 pb-6 space-y-4 justify-items-center items-end">
+    {kit.technologies.map(({  Icon }) => (
+      <div>
         { Icon && <TechIcon {Icon} />}
```

Comment:
In the figma the light mode main page designs show a slight grey background

<img width="120" alt="Screenshot 2023-06-26 at 5 13 35 PM" src="https://github.com/thisdot/starter.dev/assets/67210629/045314e3-0a98-45f5-9cc6-afcba470f9b8">

I think we should add that instead of using the white background in light mode